logger.go

```diff
@@ -3,50 +3,32 @@ package micrologger
 
 import (
 	"context"
-	"fmt"
 	"io"
-	"os"
-	"time"
 
 	kitlog "github.com/go-kit/kit/log"
-	"github.com/go-stack/stack"
 
-	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger/loggermeta"
 )
 
-// Config represents the configuration used to create a new logger.
 type Config struct {
-	// Settings.
 	Caller             kitlog.Valuer
 	IOWriter           io.Writer
 	TimestampFormatter kitlog.Valuer
 }
 
-// DefaultConfig provides a default configuration to create a new logger by best
-// effort.
-func DefaultConfig() Config {
-
-	return Config{
-		// Settings.
-		Caller: func() interface{} {
-			return fmt.Sprintf("%+v", stack.Caller(4))
-		},
-		IOWriter: os.Stdout,
-		TimestampFormatter: func() interface{} {
-			return time.Now().UTC().Format("2006-01-02 15:04:05.000")
-		},
-	}
+type KitLogger struct {
```

Comment:
I would call this `MicroLogger`.

Reply:
TBH none of those names are good. I think it should be Logger + Interface. I'll rename as you suggest.